network_simulator/common.py

```diff
@@ -25,6 +25,11 @@
 """Initial router packet id of :class:`.RouterPacket` is -1."""
 ROUTER_PACKET_DEFAULT_ID = -1
 
+"""Conversion to/from bits <-> KB, MB"""
+KILOBYTE = 8.0 * 1024
```

Comment:
Actually, I believe kilobyte is actually 1000.0 bytes (i.e. 8000.0 bits) in networking. And a megabyte is 1000.0 kilobytes. See https://en.wikipedia.org/wiki/Megabyte for the distinction between megabyte and mebibyte. You could also ask Cody to confirm (I'd recommend this).